eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/io/DbsnpVariantsWriter.java

```diff
@@ -255,23 +262,48 @@ private void writeSubmittedVariantsAndOperations(List<? extends DbsnpVariantsWra
                     .collect(Collectors.toList());
 
             if (mergedIntoList.size() > 1) {
-                throw new IllegalStateException(
-                        "Clustered variant rs" + hashAndAccession.getSecond() + " was merged into several other "
-                                + "clustered variants with the same hash: " + mergedIntoList);
+                DbsnpClusteredVariantEntity activeClusteredVariant = mongoTemplate.findById(
+                        hashAndAccession.getFirst(), DbsnpClusteredVariantEntity.class);
+
+                if (activeClusteredVariant == null || !mergedIntoList.contains(activeClusteredVariant.getAccession())) {
+                    throwSeveralInactiveMergesException(hashAndAccession, mergedIntoList, activeClusteredVariant);
+                } else {
+                    originalToNewAccessions.put(hashAndAccession, activeClusteredVariant.getAccession());
+                    logger.debug("Clustered variant rs{} was merged into several other clustered variants with the "
+                                         + "same hash: {}. The accession rs{} was chosen as "
+                                         + "replacement because it's active, i.e. present in the collection for {}",
+                            hashAndAccession.getSecond(), mergedIntoList, activeClusteredVariant.getAccession(),
```

Comment:
Will this code path be exercised for rs136611820 in EVA-1384? I did `grep "chosen as replacement" cow_9913_load_to_accessioning_20181101120650.log` and I couldn't find anything.

Reply:
yes, but only if the debug level is debug. the default is info, I think.